microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -497,7 +508,53 @@ def _configure_vdpa_devices(self, node: Node) -> None:
             node.tools[Chown].change_owner(file=PurePath(device_path), user=user)
             node.tools[Chmod].chmod(path=device_path, permission=permission, sudo=True)
 
-    def _set_data_disk(self) -> None:
+    def _get_pmem_for_block_tests(self) -> str:
```

Comment:
Will this work for 2.0 as well?

Reply:
Agree. What i meant was, in 2.0 we used some config file. I was asking if grub2-mkconfig works for 2.0

For 3.0, we use grub only anyway